src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -78,9 +80,9 @@
     };
 
     private static final Map<String, Set<String>> CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE =
-            new HashMap<>();
+            new LinkedHashMap<>();
     private static final Map<String, Set<String>> GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE =
-            new HashMap<>();
+            new LinkedHashMap<>();
```

Comment:
If the other changes resolve https://github.com/checkstyle/checkstyle/pull/9083#discussion_r553663124 , then why are these changes needed?

Reply:
They are not, thanks for catching that. Fixed.